src/pages/studyView/StudyViewPage.tsx

```diff
@@ -1016,6 +1017,15 @@ export default class StudyViewPage extends React.Component<
     }
 
     render() {
+        const isLoading =
+            this.store.queriedSampleIdentifiers.isPending ||
+            this.store.invalidSampleIds.isPending ||
+            this.body.isPending;
+        const studies =
```

Comment:
we need to check that these are not pending.

Reply:
Thanks for your feedback！I apologize for the confusion caused by the naming. Here, I only need to determine if any samples were selected, without needing the specific count.